pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tap-anvil"
-version = "0.1.5"
+version = "0.1.6"
```

Comment:
not sure if this is the only place that need to update? is there any other actions needed?

Reply:
I think that's it, we're installing from main AFAIK anyway, no?

Reply:
we do have a version number here
https://github.com/svinstech/vouch-airflow/blob/main/setup.sh#L26

Reply:
ah so we do need to update in airflow too and create the tag/release

Reply:
so that's after we merge this and then click on the `create a new release` here?
![Screenshot 2022-11-11 at 3 55 49 PM](https://user-images.githubusercontent.com/32397832/201445685-33a71a14-6113-4a00-856e-e0c7b24a2eb8.png)

Reply:
yeah!